client/src/components/EditReviewForm.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react'
+
+import ErrorList from './ErrorList'
+
+const EditReviewForm = ({ previousReview, patchReview, updateEditable, errors }) => {
+  const [review, setReview] = useState({
+    rating: previousReview.rating,
+    title: previousReview.title || '',
+    content: previousReview.content || '',
+    id: previousReview.id
+  })
```

Comment:
Setting the initial value of state with something from props can be tricky.  On the initial rendering a review may have a title of `""` but after new props are passed down from a `useEffect` being run in the parent component, the state would not change based on different props.  It looks like the rendering of this component is controlled by a conditional which is why this is working since it is only loaded when there is the correct information in the parent element